one_fm/utils.py

```diff
@@ -3153,6 +3153,58 @@ def has_super_user_role(user=None):
                 return True
     return False
 
+@frappe.whitelist()
+def get_approver(employee, date=False):
```

Comment:
The same method Defined on [line 3077](https://github.com/ONE-F-M/one_fm/blob/5c271d9743f22043d47ddf149b0bb00e4b0530a5/one_fm/utils.py#L3077)